docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/MulticlassClassification.ttinclude

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+<# if (TrainerOptions != null) { #>
+<#=OptionsInclude#>
+<# } #>
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.MulticlassClassification
```

Comment:
>Microsoft.ML [](start = 10, length = 12)

please drop Microsoft.ML prefix as per [https://github.com/dotnet/machinelearning/issues/3205](https://github.com/dotnet/machinelearning/issues/3205)

Reply:
done

---
In reply to: [272754353](https://github.com/dotnet/machinelearning/pull/3159#discussion_r272754353) [](ancestors = 272754353)